packages/lesswrong/components/ea-forum/EAHome.tsx

```diff
@@ -43,22 +43,41 @@ const getStructuredData = () => ({
   }),
 })
 
-const styles = (theme: ThemeType): JssStyles => ({
+const styles = (_theme: ThemeType) => ({
   spotlightMargin: {
     marginBottom: 24,
   },
 });
 
-const EAHome = ({classes}: {classes: ClassesType}) => {
+const FrontpageNode = ({classes}: {classes: ClassesType<typeof styles>}) => {
```

Comment:
This component being defined inline without a `useCallback` was causing unnecessary rerenders so I refactored it into a separate component